vm_supervisor/vm/firecracker/instance.py

```diff
@@ -170,15 +170,18 @@ def _create_network_file(self) -> bytes:
 
         ip = self.get_vm_ip()
         route = self.get_vm_route()
+        ipv6 = self.get_vm_ipv6()
+        ipv6_gateway = self.get_vm_ipv6_gateway()
 
         network = {
             "network": {
                 "ethernets": {
                     "eth0": {
                         "dhcp4": False,
                         "dhcp6": False,
-                        "addresses": [str(ip)],
+                        "addresses": [str(ip), ipv6],
```

Comment:
```suggestion
                        "addresses": [ip, ipv6],
```
Remove the unneeded `str` method, but we need to ensure that any IP is received